package.json

```diff
@@ -54,7 +54,7 @@
     "test": "jest",
     "clean:allureFolders": "rm -R -f allure-reports/* && rm -R -f allure-results/*",
     "beforeIntTest": "npm run clean:allureFolders",
-    "test:integration": "jest --config jest.integration.config.ts",
+    "test:integration": "jest --config jest.integration.config.ts -i",
```

Comment:
We are deleting the btm_transactions folder in storage bucket at the start of the test by calling a function called deleteDirectoryRecursiveInS3 in transaction view athena tests where as in sns-s3-tests we checks event id present in the btm_transaction folder so these tests will be interrupted when running parallel. I think Once we implement BTM-340 clean up steps for each tests than we can enable the tests to run parallely

Reply:
Only data change we can do is event time stamp if we generate event based on different month and year and then we can get the unique result when we filter by year and month for view tests but that to require clean up steps after event generated in each test run. So in BTM-340 we can implement clean up steps for events generated in others tests (except view tests) so that we no need to delete Entire S3 directory

Reply:
The reason I was saying we need clean up steps For eg for this test event   ${"IPV_PASSPORT_CRI_REQUEST_SENT"}  and vendor ${"client1"} Eventhough If I  randomly generate month (it can be 1-12) and year (as per rate table we have year date from 2022) so (2022,2023) the combinations of month and year can be reached if we run tests in some number of times.